From e8c3496df3384fe69d95e43ddbf577afcdd7c798 Mon Sep 17 00:00:00 2001
From: Stefan Hansson <newbyte@postmarketos.org>
Date: Fri, 30 Aug 2024 20:33:29 +0200
Subject: [PATCH] Regenerate Appstream metainfo

Remove use of literal ampersand symbols which causes validation errors.
---
 resources/com.jpexs.decompiler.flash.metainfo.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/resources/com.jpexs.decompiler.flash.metainfo.xml b/resources/com.jpexs.decompiler.flash.metainfo.xml
index 4a099f97e..bd914937e 100644
--- a/resources/com.jpexs.decompiler.flash.metainfo.xml
+++ b/resources/com.jpexs.decompiler.flash.metainfo.xml
@@ -92,7 +92,7 @@
 					<li>Script/Class initializers order of assignment</li>
 					<li>#2277 Return statement in initializer</li>
 					<li>Imports in script initializer</li>
-					<li>#2279 AS3 Decompilation - assignments on the right side of `&&` and `||` operators</li>
+					<li>#2279 AS3 Decompilation - assignments on the right side of `&amp;&amp;` and `||` operators</li>
 					<li>#2279 Embed assets with file base name ending with a space</li>
 					<li>Embed tag - Wav files need to be embedded in assets.swf</li>
 					<li>#2282 FLA export - visible flag</li>
-- 
2.46.0

